src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -47,13 +89,31 @@ public void SetAccessControl_DirectoryInfo_DirectorySecurity_InvalidArguments()
         }
 
         [Fact]
+        public void SetAccessControl_DirectoryInfo_DirectorySecurity()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+            DirectorySecurity directorySecurity = new DirectorySecurity();
+
+            FileSystemAclExtensions.SetAccessControl(directoryInfo, directorySecurity);
```

Comment:
Is this potentially changing access controls on the file system?  We need to be careful about that.